### PR TITLE
Add a Solus build env script.

### PR DESCRIPTION
**Pull request by ZVNexus**

### CODE REVIEW

File: setup/solus.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+printf "\nYou are now ready to build Android!"
+printf "\n\n"
+
+# CCACHE
+while true; do
+    read -p "Do you wish to enable CCACHE? (y/n)" yn
+    case $yn in
+        [Yy]* ) echo "export USE_CCACHE=1
+export CCACHE_DIR=/home/nexus/.ccache" >> /home/$(whoami)/.bashrc ; break;;
```
Comment (akhilnarang) [inline setup/solus.sh]: `/home/nexus`, nice :P
Also just use `${HOME}`

Comment (ZVNexus) [reply setup/solus.sh]: Didn't catch that, my bad.
Didn't know ${HOME} was a thing, will put it in :)

**Comment by Infect-Ed** [comment]
This is good to go.

**Comment by akhilnarang** [comment]
Thanks for testing @Infect-Ed, merged with a few changes.

**Pull request merged**